tools/c8l

```diff
@@ -676,7 +676,11 @@ chainloop_attestation_init() {
   if [ -n "${CHAINLOOP_WORKFLOW_NAME}" ]; then
     WF_NAME_VALUE="--workflow-name ${CHAINLOOP_WORKFLOW_NAME}"
   fi
-  r=$(chainloop attestation init -f --remote-state --output json $CR_VALUE $WF_NAME_VALUE)
+  PROJECT_NAME_VALUE=""
```

Comment:
@danlishka I might need your help here. I am trying to figure out how the push works.

I think it uses some kind of cache, but I guess that now we will need to update the cache to also take into account the project name, but I am struggling to find that. No rush

Reply:
ahh no, the cache is unique within the run, ok.

I am wondering then why we are passing the workflow name here

https://github.com/chainloop-dev/chainloop/blob/f215274ee36f6f0cb7010eadaeafbb60b394265c/.github/workflows/docs_deploy.yml#L96